lib/plugins/amplitude/Amplitude.ts

```diff
@@ -0,0 +1,21 @@
+import { track } from '@amplitude/analytics-node'
+import type { AmplitudeReturn, BaseEvent, Result } from '@amplitude/analytics-types'
+
+export class Amplitude {
```

Comment:
maybe AmplitudeClient for clarity?

Reply:
This class was already present, changing it will be a breaking change and I am not sure if creating a major release to change a name is worth it or not, wdyt?

Reply:
fair point, let's combine that with other breaking changes

Reply:
Yep, I agree! on next release we can combine both 🙌